composer.json

```diff
@@ -4,12 +4,15 @@
     "require": {
         "php": "^7.1.3",
         "ext-ctype": "*",
+        "ext-http": "*",
```

Comment:
Spėju, kad namų darbui tau nėra būtinas `ext-http`?..

Nes dėl to nepraeina testai ir yra sunkiau tikrinti darbus (įskaitant ir vertinimo pakėlimą po pataisymų):
![image](https://user-images.githubusercontent.com/1453957/70466579-1cf9b980-1acc-11ea-8175-237aabba9fd7.png)

Plačiau:
 * https://help.github.com/en/actions/automating-your-workflow-with-github-actions/software-installed-on-github-hosted-runners#ubuntu-1804-lts

Reply:
Pabandysiu parodyti pavyzdį, kaip galima pataisyti: https://github.com/Arnas277/kickstart/pull/1

Bet realiuose projektuose reikėtų derintis prie egzistuojančios aplinkos, o ne sakyti `pakeiskite serverį pas klientą, nes pas mane kompe veikia geriau`...